tests/providers/amazon/aws/operators/test_rds.py

```diff
@@ -601,6 +582,7 @@ def test_create_event_subscription_no_wait(self, mock_await_status):
             source_ids=[DB_INSTANCE_NAME],
             aws_conn_id=AWS_CONN,
             dag=self.dag,
+            wait_for_completion=False,
```

Comment:
From the name, this test is supposed to check the "no wait" condition, but `wait_for_completion` was set to `True`. This wasn't caught [for the above reason](https://github.com/apache/airflow/pull/27410/files#r1009685212).